zorrom/mrom.py

```diff
@@ -54,6 +78,23 @@ def __init__(self, txt=None, bin=None, verbose=False):
     def desc(self):
         return 'Unspecified'
 
+    @staticmethod
```

Comment:
These probably shouldn't have self. This was a copy and paste error on txtwh() which was then copied more